src/meta_schedule/schedule_rule/multi_level_tiling.cc

```diff
@@ -115,6 +115,7 @@ std::vector<State> MultiLevelTilingNode::ApplySubRules(std::vector<State> states
   states = SubRule(std::move(states), [&](State state) { return TileLoopNest(std::move(state)); });
   states = SubRule(std::move(states), [&](State state) { return AddWriteReuse(std::move(state)); });
   states = SubRule(std::move(states), [&](State state) { return AddReadReuse(std::move(state)); });
+  states = SubRule(std::move(states), [&](State state) { return AddAsyncPipeline(std::move(state)); });
```

Comment:
need to check target and cuda arch version

Reply:
added now